### PR TITLE
[enhance]: Update tutorials for changing training settings

### PR DESCRIPTION
**Pull request by ZwwWayne**
This PR adds tutorials for changing training settings, including optimization, learning rate scheduler.

### CODE REVIEW

**Comment by hellock** [comment]
Task linked: [CU-49ta06 Creating hooks](https://app.clickup.com/t/49ta06)

File: docs/tutorials/customize_runtime.md [hunk 1]
```diff
@@ -157,6 +157,22 @@ We support many other learning rate schedule [here](https://github.com/open-mmla
...
+```python
+workflow = [('train', 1)]
+```
+which means running 1 epoch for training.
+Sometimes user may want to check some metrics (e.g. loss, accuracy) about the model on the validate set.
+In such case, we can set the workflow as
+```python
+[('train', 1), ('val', 1)]
+```
+so that 1 epochs for training and 1 epoch for validation will be ran iteratively.
```
Comment (ZwwWayne) [inline docs/tutorials/customize_runtime.md:173]: 1. Do we need to clarify whether the model will be trained on validation set, or the workflow simply performs inference and calculate the loss?
2. In such case, what is the relation ship between this workflow and `total_epochs`, e.g., if `total_epochs=12`, will it train model on training set by 12 epochs, or perform train and val workflows by 6 epochs, respectively?

Comment (Johnson-Wang) [reply docs/tutorials/customize_runtime.md]: Speaking of that, I think we may also add some description about the equivalence about [('train', 1), ('val", 1)] and [('train', 1) but using evaluation hook.

File: docs/tutorials/customize_runtime.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+which means running 1 epoch for training.
+Sometimes user may want to check some metrics (e.g. loss, accuracy) about the model on the validate set.
+In such case, we can set the workflow as
+
+```python
+[('train', 1), ('val', 1)]
+```
+
+so that 1 epoch for training and 1 epoch for validation will be ran iteratively.
```
Comment (Johnson-Wang) [inline docs/tutorials/customize_runtime.md:177]: will be run

File: docs/tutorials/customize_runtime.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+custom_hooks = [
+    dict(type='MyHook', a=a_value, b=b_value, priority='NORMAL')
+]
+```
+
+### Modify default runtime hooks
+
+There are some common hooks that are not registerd through `custom_hooks`, they are
+
+- log_config
```
Comment (Johnson-Wang) [inline docs/tutorials/customize_runtime.md:275]: Describe the default priorities of these hooks so that the users may know what priority to use in their own hooks.

**Review by hellock — APPROVED** [review]

File: docs/tutorials/customize_runtime.md [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+
+```python
+custom_hooks = [
+    dict(type='MyHook', a=a_value, b=b_value, priority='NORMAL')
+]
+```
+
+### Modify default runtime hooks
+
+There are some common hooks that are not registerd through `custom_hooks`, they are
```
Comment (dreamerlin) [inline docs/tutorials/customize_runtime.md:275]: typo here `registered`
